### ISSUE TITLE
I can't use choosenim to remove nim 2.0.2

### ISSUE THREAD

**Issue by kayodde** [dom96/choosenim#334]
`root@vultr:~# choosenim remove --verbose 2.0.2`
`Info: No client ID found in '/root/.choosenim/analytics', not sending analytics.`
`Info: No client ID found in '/root/.choosenim/analytics', not sending analytics.`
`Info: No client ID found in '/root/.choosenim/analytics', not sending analytics.`
`Info: No client ID found in '/root/.choosenim/analytics', not sending analytics.`
`Error: /__w/choosenim/choosenim/src/choosenim.nim(350) choosenim`
`... /__w/choosenim/choosenim/src/choosenim.nim(338) performAction`
`... /__w/choosenim/choosenim/src/choosenim.nim(316) remove`
`... Cannot remove current version.`

I installed nim to the wrong user and I would like to uninstall it, but I cannot as it throws up this extremely vague error.
Please help!
I'm on debian 12

**Comment by penguinite** [comment]
You are correct in that the error is extremely vague, a better message would have been "You can't uninstall the current version, switch to a different compiler first"

You can't uninstall the current nim toolchain, if you are trying to switch to a different version of nim then switch to that version before running the uninstall message, fx. if you want to use 2.0.0 then run `choosenim 2.0.0` and then run the uninstallation command.

If you are trying to uninstall Nim (and not install any other version of it) then you can follow the instructions in this issue to uninstall choosenim: #335
